public/scripts/app.js

```diff
@@ -2,6 +2,169 @@ console.log("Sanity Check: JS is working!");
 
 $(document).ready(function(){
 
-// your code
+  console.log('app.js loaded!');
 
+  $.ajax({
+    method: 'GET',
+    url: '/api/boardgames',
+    success: renderMultipleGames
+  });
+
+// Displays new game form on click
+$('.new-game').on('click', function() {
+  $('.hidden-form').toggle();
+  $('.new-game').toggle();
+})
+
+// Allows user to go back to main view from new game form
+$('#cancel-btn').on('click', function(event) {
+  event.preventDefault();
+  $('.hidden-form').toggle();
+  $('.new-game').toggle();
+})
+
+// Creates new game on submit
+$('.new-game-form').on('submit', function(event) {
```

Comment:
This is the only longer, data-sending callback that you write inline--it would be better to be consistent, and define this method where you define handleEditGame/handleUpdateGame/etc.